lib/minify.js

```diff
@@ -1,24 +1,30 @@
 "use strict";
+/* eslint-env browser, es6, node */
+
+import {
+    defaults,
+    Dictionary,
+    HOP,
+} from "./utils.js";
+import {
+    AST_Node,
+    AST_Toplevel,
+} from "./ast.js";
+import { parse } from "./parse.js";
+import { OutputStream } from "./output.js";
+import { Compressor } from "./compress.js";
+import { base54 } from "./scope.js";
+import { SourceMap } from "./sourcemap.js";
+import {
+    mangle_properties,
+    reserve_quoted_keys,
+} from "./propmangle.js";
 
 var to_ascii = typeof atob == "undefined" ? function(b64) {
-    if (Buffer.from && Buffer.from !== Uint8Array.from) {
-      // Node >= 4.5.0
-      return Buffer.from(b64, "base64").toString();
-    } else {
-      // Node < 4.5.0, old API, manual safeguards
-      if (typeof b64 !== "string") throw new Errror("\"b64\" must be a string");
-      return new Buffer(b64, "base64").toString();
-    }
+    return Buffer.from(b64, "base64").toString();
 } : atob;
 var to_base64 = typeof btoa == "undefined" ? function(str) {
-    if (Buffer.from && Buffer.from !== Uint8Array.from) {
-      // Node >= 4.5.0
-      return Buffer.from(str).toString("base64");
-    } else {
-      // Node < 4.5.0, old API, manual safeguards
-      if (typeof str !== "string") throw new Errror("\"str\" must be a string");
-      return new Buffer(str).toString("base64");
-    }
+    return Buffer.from(str).toString("base64");
```

Comment:
Since 6.x we can use arrow functions
`str => Buffer.from(str).toString("base64")`